### PR TITLE
#15717: Add MEGAcmd update event

### PR DESCRIPTION
**Pull request by aabellagm**
Check if MEGAcmd has been updated and:
- Show a message.
- Send a MEGAcmd update event.

### CODE REVIEW

File: src/configurationmanager.cpp [hunk 1]
```diff
@@ -624,11 +631,29 @@ void ConfigurationManager::loadConfiguration(bool debug)
...
+            fiVer >> latestVersion;
+            fiVer.close();
+        }
+
+        if (latestVersion.size()>0 && (MEGACMD_CODE_VERSION > stol(latestVersion)))
+        {
+            hasBeenUpdated = true;
+            if (debug)
+                cout  << "MEGAcmd has been updated." << endl;
+        }
```
Comment (polmr) [inline src/configurationmanager.cpp:653]: Picky comments to follow coding style (although surrounding one wasn't following it ;-) ):
preferable: `latestVersion.size()>0` -> `!latestVersion.empty()`, or `latestVersion.size()`   or `latestVersion.size() > 0` in any case.
`cout` -> `std::cout`, same for endl
and surround the if body-code between '{' and '}

File: src/configurationmanager.cpp [hunk 2]
```diff
@@ -51,6 +52,7 @@ bool is_file_exist(const char *fileName)
 }
 
 string ConfigurationManager::configFolder;
+bool ConfigurationManager::hasBeenUpdated=false;
```
Comment (polmr) [inline src/configurationmanager.cpp:55]: `ConfigurationManager::hasBeenUpdated=false;` -> `ConfigurationManager::hasBeenUpdated = false;`

File: src/configurationmanager.cpp (hunk 1 above)
Comment (wadh73888) [reply src/configurationmanager.cpp]: 364bee66909dfda784e27639f1d6b96d4bc7ae37

Comment (wadh73888) [reply src/configurationmanager.cpp]: > Picky comments to follow coding style (although surrounding one wasn't following it ;-) ):
> preferable: `latestVersion.size()>0` -> `!latestVersion.empty()`, or `latestVersion.size()`   or `latestVersion.size() > 0` in any case.
> `cout` -> `std::cout`, same for endl
> and surround the if body-code between '{' and '}

**Pull request merged**